src/gam.py

```diff
@@ -7867,7 +7867,8 @@ def doShowServiceAccountKeys():
     print('{0}: {1}'.format(parts[i][:-1], parts[i+1]))
   for key in keys:
     key['name'] = key['name'].rsplit('/', 1)[-1]
-    key['current'] = key['name'] == currentPrivateKeyId
+    if key.get('keyType', '') == 'USER_MANAGED' and key['name'] == currentPrivateKeyId:
```

Comment:
no reason to check if it's user managed here and we may use GOOGLE_MANAGED in the future, thus no reason to do this check. Also, let's make showing all keys the default.